html/assets/css/index.css

```
@@ -29,4 +29,8 @@
 .extension-view-project {
 	margin-left: 7px;
 	margin-right: 7px;
+}
+
+.tile {
+	align-items: center;
```

Instead of `align-itens` try:

```suggestion
	justify-content: center;
```

---

This changes doesn't effect anything. Weird...

---

There's some weird thing occurring between our browsers. See the fiddle implementation and see if it changes anything.